runtime-modules/working-group/src/benchmarking.rs

```diff
@@ -951,46 +899,16 @@ benchmarks_instance! {
                 1,
                 description,
                 OpeningType::Regular,
-                Some(stake_policy),
+                stake_policy,
                 Some(BalanceOf::<T>::max_value())
             ).into()
         );
     }
 
```

Comment:
Why did you remove this branch? Also, in "leave_role" benchmark you could use "_" syntax extrinsic call.

Reply:
Because since the stake is now required there is no way to leave the role immediatly, which also means I should remove that branch from the extrinsic(Since no working group should have `MinimumStakeForOpening == 0` since that would prevent the stake from being an anti-spam mechanism and we require the leaving period for any application to be greater than `MinUnstakingPeriod` which means that no application can have an unstaking period of 0)